packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -2133,10 +2213,8 @@ describe("AttributesController", function () {
                 }
             });
 
-            await TestUtil.expectThrowsAsync(
-                consumptionController.attributes.getSharedVersionsOfRepositoryAttribute(relationshipAttribute.id),
-                "error.consumption.attributes.invalidPropertyValue"
-            );
+            const result = await consumptionController.attributes.getSharedVersionsOfRepositoryAttribute(relationshipAttribute.id);
+            expect(result).toHaveLength(0);
```

Comment:
This test "should throw", according to its name.

Help me out here. What changed so that the call to `getSharedVersionsOfRepositoryAttribute` doesn't raise an exception anymore?

Reply:
Ah right, I need to rename it. We also allow other Attributes than RepositoryAttributes as input (function and use case will need renaming), since we didn't consider ThirdPartyRelationshipAttributes at the time of implementing it. Now, we need its functionality in the ReadAttributeRequestItemProcessor line 88, which should also work with ThirdPartyRelationshipAttributes in addition to RepositoryAttributes.